src/config/algob.config.ts

```diff
@@ -0,0 +1,10 @@
+import algosdk from "algosdk";
```

Comment:
Why we need algob.config.ts?

Reply:
because webmode currently doesn't support query localstate and globalstate so to get those values I use algodclient

Reply:
I think we don't need it. The network user use on `Algosigner` and in `algob.config.ts` can be **different** and this is big problem.

We can use [algosigner.algod](https://github.com/PureStake/algosigner/blob/develop/docs/dApp-integration.md#algosigneralgod-ledger-mainnettestnet-path-algod-v2-path--) in this case and this method ensure we use the same client to query data and send txn to network.
 
You can query globalstates and localstates of application by the endpoint `GET /v2/applications/{application-id}`. More details at [here](https://developer.algorand.org/docs/rest-apis/algod/v2/#get-v2applicationsapplication-id)

Reply:
thanks for the advice, I didn't thought that way